Makefile

```diff
@@ -1,13 +1,19 @@
 examples: minimal basic modern animation
 
+INCLUDES := `python-config --includes`
```

Comment:
Actually, I'm not sure about this. This Makefile was intended to be pedagogical, i.e. to demonstrate a complete command-line that is sufficient to compile working examples, and to assure potentially interested users that python is really the only needed dependency.

I feel like adding an indirection through `python-config` goes against these goals, even if I agree that it would be better style.

Reply:
`python-config` is part of `python-dev` which is a dependency anyway, so not sure why that is a problem. The call to `python-config` is required to pick the correct version of Python.